test/format-impl-test.cc

```diff
@@ -523,3 +523,37 @@ TEST(format_impl_test, to_utf8) {
   EXPECT_EQ(s, u.str());
   EXPECT_EQ(s.size(), u.size());
 }
+
+FMT_CONSTEXPR20 bool constexpr_iceil() {
+  for (double v : std::initializer_list<double>{
+           ((std::numeric_limits<int>::min)() + 0.5),
+           -1.2,
+           -0.2,
+           0.0,
+           0.2,
+           1.2,
+           4.0,
+           ((std::numeric_limits<int>::max)() - 0.5),
+       }) {
+    auto r = fmt::detail::iceil(v);
+    fmt::detail::ignore_unused(r);
+  }
+  return true;
+}
```

Comment:
Once the branching on constant evaluation is removed, we won't need this anymore.